README.md

```diff
@@ -80,6 +80,11 @@ An `nvmrc` file is included if using [nvm](https://github.com/creationix/nvm).
 ## Development
 default port 3002
 
+**PostGres**
+install and start Postgres, include login information for it per PG* variables in a .env file in root directory per env.sample
+once Postgres is installed run `yarn install` to install necessary packages.
```

Comment:
Why did you switch from npm to yarn midstream?

Reply:
I just want to use it a bit for practice. Also, it has a cat as it's icon, and a few of my friends in webdev insist its better.